### PR TITLE
Change doc string for Axes.arrow()

### PR DESCRIPTION
**Pull request by DumbMachine**
Changed the string from "The x/y-coordinate of the arrow base" to "The x and y coordinates of the arrow base".

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -4795,7 +4795,7 @@ def arrow(self, x, y, dx, dy, **kwargs):
         Parameters
         ----------
         x, y : float
-            The x/y-coordinate of the arrow base.
+            The x and y coordinates of the arrow base.
         dx, dy : float
             The length of the arrow along x/y-direction.
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py]: If we remove `x/y` notation, please change this as well.

Comment (DumbMachine) [reply lib/matplotlib/axes/_axes.py]: Done. Would you recommend changing it in other places like [here](https://github.com/DumbMachine/matplotlib/blob/patch-1/lib/matplotlib/axes/_axes.py#L1660)

**Comment by jklymak** [comment]
Can we edit the title of this PR to something that doesnt reference source code line numbers?  Otherwise looks great!   Thanks.

**Comment by dstansby** [comment]
👍 thanks a lot for the fix!

**Pull request merged**